.github/workflows/PR_summary.yml

```diff
@@ -41,10 +41,8 @@ jobs:
 
         # Checkout the merge base
         git checkout "$(git merge-base master ${{ github.sha }})"
-        # git checkout origin/master scripts/count-trans-deps.py scripts/import-graph-report.py
-        # get the files that we need from the current PR
-        # TODO ***change*** to `origin/master` in the line below, when merging or right after
-        git checkout origin/adomani/move_decls_autocomment scripts/update_PR_comment.sh
+        # get the files that we need from master
+        git checkout origin/master scripts/update_PR_comment.sh scripts/count-trans-deps.py scripts/import-graph-report.py
```

Comment:
```suggestion
```
Can these lines be removed entirely?  If a PR triggers the bot, it should have merged a version of master that contains the scripts anyway.  Will Git use an earlier master for its comparisons?

Reply:
I think you are right.